cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -299,16 +311,17 @@ def create_settings(self):
 
 -  *%(FI_AUTOMATIC)s:* The size is computed as 1/40 the size of the
    image or 30 pixels, whichever is smaller.
--  *%(FI_OBJECT_SIZE)s:* The size is obtained relative to the width of
-   artifacts to be smoothed.
--  *%(FI_MANUALLY)s:* Use a manually entered value.
+-  *%(FI_OBJECT_SIZE)s:* The module will calculate the smoothing size
+   based on the width of artifacts to be smoothed.
+-  *%(FI_MANUALLY)s:* You can enter a value yourself.
 """ % globals())
 
         self.object_width = cps.Integer(
             "Approximate object size", 10, doc="""\
-*(Used only if %(FI_AUTOMATIC)s is selected for smoothing filter size calculation)*
+*(Used only if %(FI_OBJECT_SIZE)s is selected for smoothing filter size calculation)*
 
-Enter the approximate width of the artifacts to be smoothed, in pixels.
+Enter the approximate width of the largest typical object, in pixels.
+The smoothing size will be set to preserve objects of that size.
```

Comment:
I don't believe that this is correct, I believe this is the size of things you wish to smooth out.  Note it calls to `%(FI_OBJECT_SIZE)s`, whose definition is 
```
The module will calculate the smoothing size  based on the width of artifacts to be smoothed.
```

Reply:
Ok, I am nearly certain that %(FI_AUTOMATIC)s ought to have been changed to %(FI_OBJECT_SIZE)s so I think we are all set on that. 

As for how it's described, yeah, I think the documentation has been unclear about whether we are talking about the size of objects (that you want to preserve) or artifacts (that you want to capture as the illum function). I made this one change but if it's correct we'd need to change the other line you refer to as well (and should check the rest of text too). 

Does anyone know for certain what the code is actually doing and if it seems more like the size of objects (that you want to preserve) or artifacts (that you want to capture as the illum function)?

Reply:
My reading of the documentation is minimum size of objects to preserve/maximum size of artifacts to capture (remove or ignore). I haven't had the time to sit down with the code, but generally the smoothing filter size determines the size of features to keep. Features larger than the smoothing filter size are generally preserved.